multi-node/aws/pkg/config/config.go

```diff
@@ -116,7 +120,15 @@ const (
 
 func (c Cluster) Config() (*Config, error) {
 	config := Config{Cluster: c}
-	config.ETCDEndpoints = fmt.Sprintf("http://%s:2379", c.ControllerIP)
+	etcdEndpoints := strings.Split(c.EtcdEndpoints, ",")
+	for i := range etcdEndpoints {
+		_, err := url.Parse(etcdEndpoints[i])
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
I think this validation should be in config.valid(), as well as a check that etcdEndpoints isn't empty, since we do do `etcdEndpoints[0]` below.